trace-model/run.sh

```diff
@@ -2,5 +2,14 @@
 
 source docker.properties
 
+base_path=`realpath .`
```

Comment:
This is because the directory path can be different based on the DLAMI or vanilla AMI being used to build this